Dermveda/dv-components
# Style: Card Atom - maxWidth responsive breakpoints

This PR solves the issue when there is an unbalanced number of cards in the CardRow component when showing all and the last row displayed larger cards because of the flex property.

**Steps to Test:**

- Run Storybook.
- Go to Organisms > Templates > CardRow.
- Under knobs, check showAll.
- In large/xl view, expect that the last 2 cards are in their own row and match the size of the cards in the first row.
- In tablet view, expect that the cards are wide in width (500px).
- In mobile view, expect that the cards are wide in width (500px).
